jina/resources/helloworld.reduce.yml

```diff
@@ -20,6 +20,6 @@ requests:
           start: 0
           end: 50
           granularity_range: [0, 0]
-          adjacency_range: [0, 1]
+          adjacency_range: [0, 0]
```

Comment:
This is something related to the unit test I proposed. Setting both ranges to `[0, 0]` means applying to neither granularity nor adjacency. The document should stay intact. But the following test failed.

```
def test_both_with_0_0():
    docs = build_docs()
    driver = SliceQL(
        start=0,
        end=1,
        adjacency_range=(0, 0),
        granularity_range=(0, 0),
        recur_on=["chunks", 'matches']
    )
    print(f'len(docs): {len(docs)}')
    driver._traverse_apply(docs)
    assert len(docs) == DOCUMENTS_PER_LEVEL  # this failed unexpectedly
    assert len(docs[0].chunks) == DOCUMENTS_PER_LEVEL
    assert len(docs[0].chunks[0].matches) == DOCUMENTS_PER_LEVEL
    assert len(docs[0].matches) == DOCUMENTS_PER_LEVEL
    assert len(docs[0].matches[0].chunks) == DOCUMENTS_PER_LEVEL
    assert len(docs[0].matches[0].matches) == DOCUMENTS_PER_LEVEL
    assert len(docs[0].matches[0].matches[0].chunks) == DOCUMENTS_PER_LEVEL
```